### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by rtgoodwin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+
+The tool source is OS independent (written in Go) and binaries are available for Windows, MacOS, and Linux. 
+
+# How it works
+
+*cs-reboot-info* queries the Rackspace Cloud US and UK cloud infrastructures (both First and Next Generation). It identifies any Cloud Servers with a metadata key named *"rax:reboot_window"*. This key carries a value that shows the start and end times of the scheduled reboot window for the Cloud Server. 
+
+The format of the metadata:
+
+| Key | Value  (example)|
```
Comment (smashwilson) [inline README.md]: Want to put this in triple-backticks so it's source-formatted?

**Comment by smashwilson** [comment]
Also, is this going to collide with #2?

**Comment by rtgoodwin** [comment]
This supersedes #2

**Comment by smashwilson** [comment]
Just going to merge as-is, we can fix later.

**Pull request merged**